src/factories/request.js

```diff
@@ -101,6 +101,10 @@ class RequestFactory {
           headers['X-MOLTIN-CURRENCY'] = config.currency
         }
 
+        if (config.language) {
```

Comment:
@RyanKirby89 if I am understanding correctly this would send the specified language header on all requests? could the user still pass in a language code on a specific request? 

My concern would be I think only product requests listen for this header (not a big deal), also what if the customer is doing something multilingual and needs to request stuff in different languages

Reply:
@samblacklock Yeah, good point. I dont think there is necessarily anything wrong with setting a header for all endpoints, as it will just be ignored by the api for those endpoints that dont need it - however I think we should have the ability to pass in custom headers for a particular endpoint - not sure this is currently supported though, so maybe worth creating an additional ticket for custom headers in single requests? @shaunmaharaj - what do you think?

Reply:
Then we are giving people the option of wether they want to set the header in all requests, or just in the product requests?

Reply:
@samblacklock I suppose I could move the headers into the send function instead, and do a check before sending the request, but then I would have to then also remove the language header from the header after the request is sent, which seems a bit blah, but that would be an option if we only wanted to send them in the product service. It just means that if we ever extended this functionality out to anything else, we would have to do the same, which seems not great - eg - using it in flows etc.

Reply:
I think you both had a good point about passing custom headers in requests. It may become a requirement in the future to pass additional configurations to the back-end, so good for us to have eventually. I’ll create a new ticket for it and we can implement when we have time. For now, the implementation here looks great to me! Thanks for the work on this @RyanKirby89 & @samblacklock 😁

Reply:
@RyanKirby89 yeh I agree it sounds like a bit of a hack, and sending it on all requests isn't really a big issue. If there isn't an elegant way to specify/override the language header for products requests just yet, then maybe we stick this in for now so we at least provide some language functionality, then revisit it later?